src/openzaak/components/catalogi/admin/utils.py

```diff
@@ -202,6 +206,10 @@ def import_zaaktype_for_catalogus(
 
                 for entry in data:
                     if resource == "ZaakType":
+                        if identificatie_prefix:
+                            entry[
+                                "identificatie"
+                            ] = f"{identificatie_prefix}_{entry['identificatie']}"
```

Comment:
What happens if the length of the field exceeds max length (it's only 40)? Could you please create a unittest for it?

Reply:
Apparently its actually 50. I've just truncated the string and added a unit test for this.

Reply:
Let's discuss it. For example you have 2 zaken to import with identificaties that take 30 char, for example "SOME-GEMEENTE--2023-0000000001" and "SOME-GEMEENTE--2023-0000000002". You set prefix which has 25 char length. 
Now the last part of identificaties would be truncates and now all zaken would have the same identificaties. So the import would fail with `overlap` error, which is not very clear for user how to fix

I think it's better to check if the length of the new prefixed identificatie is longer than max. And raise 400 with the message that the combined identificatie is too long.